_data/tools.json

```diff
@@ -1934,5 +1934,14 @@
       "platforms": "Linux, MacOS, Windows",
       "note": "A vulnerability scanner for container images and filesystems.",
       "type": "SAST"
+   },
+    {
+      "title": "OWASP IDE VulScanner",
+      "url": "https://destinjidee.com/owasp-ide-vulscanner",
+      "owner": "DestinJiDee LTD",
+      "license": "Commercial or Free",
```

Comment:
If it's an OWADP project it can't be commercial

Reply:
Agreed and updated